safe-global/safe-wallet-web
# fix: Include Filled percentage for swap limit orders

## What it solves

Resolves [Notion issue](https://www.notion.so/safe-global/Limit-order-7902af48b3394a94b18d03493ae6a530)

## How this PR fixes it

- Displays `SwapProgress` as part of `SwapOrder` if its a limit swap
- Adjusts the props of `SwapProgress` to receive a `SwapOrder` object
- Updates the Storybook for `SwapProgress`
- Adds a `SwapOrder` and `OrderToken` builder and tests

## How to test it

1. Create a limit swap (that can only be partially filled)
2. Execute the transaction
3. Observe the filled percentage shown in the history

## Screenshots
<img width="867" alt="Screenshot 2024-04-22 at 15 57 59" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/34cc4bfb-5aed-40b1-b299-da0c5a0fa2b7">



## Checklist
